core/opendaq/reader/tests/test_multi_reader.cpp

```diff
@@ -1212,7 +1207,20 @@ TEST_F(MultiReaderTest, EpochChangedBeforeFirstData)
     void* valuesPerSignal[NUM_SIGNALS]{values[0], values[1], values[2]};
     void* domainPerSignal[NUM_SIGNALS]{domain[0], domain[1], domain[2]};
 
-    SizeT count{SAMPLES};
+    SizeT count{1};
+    MultiReaderStatusPtr status = multi.readWithDomain(valuesPerSignal, domainPerSignal, &count);
+    ASSERT_EQ(status.getReadStatus(), ReadStatus::Event);
+    ASSERT_TRUE(status.getEventPackets().assigned());
+    ASSERT_EQ(status.getEventPackets().getCount(), 3u);
+    ASSERT_EQ(status.getEventPackets()[0], nullptr); 
```

Comment:
Should signal "1" not produce an event packet in reader status?

Reply:
Now we have a dictionary with one signal:)